tests/Unit/Evolution/SlopeLimiters/Test_Minmod.cpp

```diff
@@ -0,0 +1,1083 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <algorithm>
+#include <array>
+#include <cstddef>
+#include <functional>
+#include <limits>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Direction.hpp"
+#include "Domain/Element.hpp"
+#include "Domain/ElementId.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Domain/Mesh.hpp"
+#include "Domain/Neighbors.hpp"
+#include "Domain/OrientationMap.hpp"
+#include "Evolution/SlopeLimiters/Minmod.hpp"
+#include "NumericalAlgorithms/LinearOperators/Linearize.hpp"
+#include "NumericalAlgorithms/LinearOperators/MeanValue.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+#include "tests/Unit/TestCreation.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+// IWYU pragma: no_forward_declare SlopeLimiters::Minmod
+// IWYU pragma: no_forward_declare Tensor
+
+namespace {
+struct scalar : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept { return "Scalar"; }
+};
+
+template <size_t VolumeDim>
+struct vector : db::SimpleTag {
+  using type = tnsr::I<DataVector, VolumeDim>;
+  static std::string name() noexcept { return "Vector"; }
+};
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.SlopeLimiters.Minmod.Options",
```

Comment:
Should you test creation with the TvbmConstant option as well?

Reply:
I do, just below. But perhaps I misunderstand...?

Reply:
I just missed it, sorry :( All good!